### ISSUE TITLE
Minify checkout.js script

### ISSUE THREAD

**Issue by trainerbill** [paypal/paypal-checkout#264]
1.  Move to webpack 2 so that we can build sourcemaps for minified files
2.  Change checkout.js to be minified.  Provide a .debug file that is unminified
3.  Test out converting stack trace errors using sourcmaps.  should be able to import checkout.js and https://github.com/novocaine/sourcemapped-stacktrace throw the error and convert it.

**Comment by kentcdodds** [comment]
ping, any progress on this? How can we help?

**Comment by trainerbill** [comment]
@kentcdodds  https://www.paypalobjects.com/api/checkout.min.js

**Comment by bluepnume** [comment]
Going to close this in favour of https://github.com/paypal/paypal-checkout/issues/44 as there's a lot more context there.
 
TLDR:

- We expose https://www.paypalobjects.com/api/checkout.min.js but https://www.paypalobjects.com/api/checkout.js is still unminified by default
- Reason being we do a helluvalot of debugging in older version of IE where sourcemap support is there, but really flaky.
- Would really like a way to minify checkout.js and flip it to unminified mode on demand for a given merchant on paypalobjects, so we get the best of both worlds.

**Comment by kentcdodds** [comment]
Glad to see it's being worked on in #44. Alex is right, and PayPal's image (as are mobile users) by the current state of things, so thanks for working on it! <3

**Comment by kentcdodds** [comment]
One thing that would be nice is if in the README the existence of the minified version was mentioned with a link to #44 to describe the trade-offs.
